raiden/api/python.py

```diff
@@ -173,20 +173,21 @@ def get_channel(
         partner_address: Address,
     ) -> NettingChannelState:
         if not is_binary_address(token_address):
-            raise InvalidAddress("Expected binary address format for token in get_channel")
+            raise InvalidBinaryAddress("Expected binary address format for token in get_channel")
 
         if not is_binary_address(partner_address):
-            raise InvalidAddress("Expected binary address format for partner in get_channel")
+            raise InvalidBinaryAddress("Expected binary address format for partner in get_channel")
 
         channel_list = self.get_channel_list(registry_address, token_address, partner_address)
         assert len(channel_list) <= 1
 
         if not channel_list:
-            raise ChannelNotFound(
-                "Channel with partner '{}' for token '{}' could not be found.".format(
-                    to_checksum_address(partner_address), to_checksum_address(token_address)
-                )
+            msg = (
+                f"Channel with partner '{to_checksum_address(partner_address)}' "
+                f"for token '{to_checksum_address(token_address)}' could not be "
```

Comment:
`token_address` is the address of the ERC20 token, the APIs are usually using that as input value.

`token_network_address` is the address of the `TokenNetwork` which is backed up by a ERC20 token. We have to use this value inside Raiden because the channel is tied to a token network.